lib/active_model/serializer.rb

```diff
@@ -167,5 +167,25 @@ def self.get_serializer_for(klass)
         end
       end
     end
+
+    # Transforms an inclusion hash into an array of corresponding existing associations,
+    # and corresponding inclusion hashes.
+    # @param [Hash] includes
+    # @return [Array] an array of pairs [association, include_hash] for matching associations
+    def expand_includes(includes)
```

Comment:
This one has a different purpose. I'm not happy with the name of this method, because what it really does is lookup each included association and returns the corresponding `Association` and include hash. It just happens to expand stuff because at this stage we can't avoid it anymore.